src/include/cpu/rpp_cpu_common.hpp

```diff
@@ -681,7 +748,8 @@ inline RppStatus resize_kernel_host(T* srcPtr, RppiSize srcSize, T* dstPtr, Rppi
         Rpp32f wRatio = (((Rpp32f) (dstSize.width - 1)) / ((Rpp32f) (srcSize.width - 1)));
         Rpp32f srcLocationRow, srcLocationColumn, pixel;
         Rpp32s srcLocationRowFloor, srcLocationColumnFloor;
-        T *srcPtrTemp, *dstPtrTemp, *srcPtrTopRow, *srcPtrBottomRow;
+        T *srcPtrTemp, *srcPtrTopRow, *srcPtrBottomRow;
```

Comment:
Codacy found an issue: [The scope of the variable 'srcPtrBottomRow' can be reduced.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/rpp/pullRequest?prid=5976402)